Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/appsettings.json

```diff
@@ -1,40 +1,56 @@
 {
-    "Logging": {
-        "LogLevel": {
-            "Default": "Warning"
-        }
-    },
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning"
+    }
+  },
 
-    "AllowedHosts": "*",
+  "AllowedHosts": "*",
 
   "ConnectionStrings": {
     "DefaultConnection": ""
   },
 
-    "IdentityServerHandlers": [
-        {
-            "AuthenticationScheme": "",
-            "ApiName": "",
-            "AuthorityUrl": "",
-            "Proxy": {
-                "Address": ""
-            },
-            "RequireHttpsMetadata": true
-        }
-    ],
+  "KormSettings": {
+    "DefaultConnection": {
+      "AutoMigrate": true
+    }
+  },
+
+  "IdentityServerHandlers": [
```

Comment:
Please remove this section. In downstream services we don't use identity server authentification, but our authorization service.